library/cipher_wrap.c

```diff
@@ -576,6 +578,7 @@ static int gcm_aes_setkey_wrap(void *ctx, const unsigned char *key,
     return mbedtls_gcm_setkey((mbedtls_gcm_context *) ctx, MBEDTLS_CIPHER_ID_AES,
                               key, key_bitlen);
 }
+#endif /* MBEDTLS_GCM_C */
 
 static const mbedtls_cipher_base_t gcm_aes_info = {
```

Comment:
From the CI, `all.sh build_tfm`:
```
cipher_wrap.c:583:36: error: unused variable 'gcm_aes_info' [-Werror,-Wunused-const-variable]
static const mbedtls_cipher_base_t gcm_aes_info = {
                                   ^
1 error generated.
Makefile:312: recipe for target 'cipher_wrap.o' failed
```

I think any configuration without GCM would have that problem.